src/connectors/openAI/mod.ts

```diff
@@ -0,0 +1,13 @@
+/** This file was autogenerated. Follow the steps in src/schema/slack/functions/README.md to rebuild **/
+import GetChatGptResponse from "./functions/get_chatGPT_response.ts";
+
+const OpenAi = {
```

Comment:
Should we update the generator so that compound acronym service names (word+acronym) appear as expected (e.g., `OpenAI`) instead of defaulted to CamelCase?

Reply:
Good question 💯  We've tried to look into this before, turns out it is complex to implement something like this in a scalable way, the simple solution is to follow CamelCase standards